src/components/Dragons/DragonItem.jsx

```diff
@@ -0,0 +1,50 @@
+// Hooks
+import { useDispatch } from 'react-redux';
+// props
+import PropTypes from 'prop-types';
+
+// React-bootstrap components
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import Badge from 'react-bootstrap/Badge';
+
+// Redux Actions
+import { reserveDragon,  cancelReserveDragon} from '../../Redux/dragons/dragons';
+
+
+const DragonItem = ({id, name, type, description, reserved, img}) => {
+  const dispatch = useDispatch();
+
+  return (
+    <li>
+      <Card className="flex-md-row border-0 mb-5">
+        <Card.Img style={{ maxWidth: '18rem', height: "18rem" }} variant="left" src={img} />
+        <Card.Body>
+          <Card.Title>{name}</Card.Title>
+          <Card.Text>{type}</Card.Text>
+          <Card.Text>
+            {(reserved) ? <Badge bg="info" className="me-2">Reserved</Badge> : ""}
+            {description}
+          </Card.Text>
+          {
+          (reserved) ? 
+            <Button variant="btn btn-outline-secondary" onClick={() => dispatch(cancelReserveDragon(id))}>Cancel Reservation</Button>
+            : 
+            <Button variant="primary" onClick={() => dispatch(reserveDragon(id))}>Reserve Dragon</Button>
+          }
+        </Card.Body>
+      </Card>
+    </li>
+  );
+}
```

Comment:
- Hey, great job team. I should have marked this optional but kindly bear with me. I'm getting a lot of eslint errors when I run your app locally on my machine. For some reason, it doesn't seem to throw the error here on GitHub but it does locally. Please let's fix this error to help us write clean and maintainable code. I've fixed the issues. Please, kindly do the update proposed below to fix it. :pray: 

- Kindly update the lines of code with these

```
// Redux Actions
import { reserveDragon, cancelReserveDragon } from '../../Redux/dragons/dragons';

const DragonItem = ({
  id, name, type, description, reserved, img,
}) => {
  const dispatch = useDispatch();

  return (
    <li>
      <Card className="flex-md-row border-0 mb-5">
        <Card.Img style={{ maxWidth: '18rem', height: '18rem' }} variant="left" src={img} />
        <Card.Body>
          <Card.Title>{name}</Card.Title>
          <Card.Text>{type}</Card.Text>
          <Card.Text>
            {(reserved) ? <Badge bg="info" className="me-2">Reserved</Badge> : ''}
            {description}
          </Card.Text>
          {
          (reserved)
            ? <Button variant="btn btn-outline-secondary" onClick={() => dispatch(cancelReserveDragon(id))}>Cancel Reservation</Button>
            : <Button variant="primary" onClick={() => dispatch(reserveDragon(id))}>Reserve Dragon</Button>
          }
        </Card.Body>
      </Card>
    </li>
  );
};
```